app/api/v2/models/auth.py

```diff
@@ -25,6 +28,20 @@ def get_user_by_email(self):
         con.close()
         return response
 
+    def user_is_admin(self):
+        conn, response = self.db, None
+        cur = con.cursor(cursor_factory=RealDictCursor)
+        try:
+            cur.execute("select role from users WHERE user_id='{}'"
+                        .format(self.user_id))
+            response = cur.fethone()
+            if response == 2:
+                return True
+            else:
+                return False
+        except Exception as e:
+            prin(e)
```

Comment:
print is misspelled, please just remove it and add another way to handle exceptions